kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -168,13 +176,15 @@ export default [
       }
       showTopicsTopic(store, { id: toRoute.params.id, pageName: toRoute.name });
     },
+    component: TopicsPage,
   },
   {
     name: PageNames.TOPICS_CONTENT,
     path: '/topics/c/:id',
     handler: toRoute => {
       showTopicsContent(store, toRoute.params.id);
     },
+    component: LearnImmersiveLayout,
```

Comment:
This component seems to be used only on this route now and its purpose isn't to provide layout anymore so I'd suggest renaming it to something like `TopicsContentPage`